Layouts.md

```diff
@@ -1746,24 +1521,24 @@ The class representing an audio track. Instances of this class are returned by t
 
 -  `file_name` - Get/set the audio track filename.
 -  `volume` - Get/set the volume of played audio track. Range is `[0...100]`
--  `playing` - Get/set whether the audio track is currently playing (boolean).
--  `loop` - Get/set whether the audio track should be looped (boolean).
+-  `playing` - [boolean] Get/set whether the audio track is currently playing.
+-  `loop` - [boolean] Get/set whether the audio track should be looped.
 -  `pitch` - Get/set the audio track pitch (float). Default value is `1`.
 -  `x` - Get/set the x position of the audio track. Default value is `0`.
 -  `y` - Get/set the y position of the audio track. Default value is `0`.
 -  `z` - Get/set the z position of the audio track. Default value is `0`.
 -  `duration` - Get the audio track duration (in milliseconds).
 -  `time` - Get the time that the audio track is current at (in milliseconds).
--  `vu` - Get the current VU meter value in mono. Range is `[0.0...1.0]`.
--  `vu_left` - Get the current VU meter value for the left audio channel. Range is `[0.0...1.0]`.
--  `vu_right` - Get the current VU meter value for the right audio channel. Range is `[0.0...1.0]`.
--  `fft` - Get the Fast Fourier Transform data for mono audio as an array of 32 values. Range is `[0.0...1.0]`.
--  `fft_left` - Get the Fast Fourier Transform data for the left audio channel as an array of 32 values. Range is `[0.0...1.0]`.
--  `fft_right` - Get the Fast Fourier Transform data for the right audio channel as an array of 32 values. Range is `[0.0...1.0]`.
+-  `vu` 🔶 - Get the current VU meter value in mono. Range is `[0.0...1.0]`.
```

Comment:
Not necessarily requires a diamond since the whole fe.Music is marked as a new class